package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "cse110-fa22-group23",
```

Comment:
Please delete this file. While you are at it, can you add it to the gitignore. I also make the mistake of using npm in the root which creates this file when you are supposed to only run it in src